apps/client/src/features/app-settings/panel/project-panel/ProjectData.tsx

```diff
@@ -86,6 +120,31 @@ export default function ProjectData() {
               {...register('title')}
             />
           </label>
+          <label>
```

Comment:
can we use an upload card similar to what we have in the sheets page?

Reply:
![image](https://github.com/user-attachments/assets/7b9284dd-18ab-4f95-aa60-3c965a76c4f4)